Examples/WebViewExample/WebViewExample/Utilities/BNCSwiftLog.h

```diff
@@ -0,0 +1,48 @@
+//
+//  BNCSwiftLog.h
+//  WebViewExample
+//
+//  Created by Jimmy Dee on 4/7/17.
+//  Copyright © 2017 Branch Metrics. All rights reserved.
+//
+
+@import Foundation;
+
+/**
+ * Swift-friendly Obj-C wrapper for BNCLogMessageInternal
+ */
+@interface BNCSwiftLog : NSObject
```

Comment:
This entire wrapper could be a single method that also takes a BNCLogLevel enum argument, like BNCLogMessageInternal itself. This was adapted from something else where this approach made more sense.